src/main/java/org/folio/rest/camunda/service/CamundaApiService.java

```diff
@@ -54,6 +54,8 @@ public Workflow deployWorkflow(Workflow workflow, String tenant) throws Workflow
         Bpmn.writeModelToStream(System.out, modelInstance);
       }
 
+      logger.error("Failed to deploy workflow {}:{}. {}", workflow.getId(), workflow.getName(), e.getMessage(), e);
```

Comment:
Why is there an additional logger when the exception is already being thrown?

Reply:
Because not all the possible exceptions have an exception handler. 

`NotFoundException | NotValidException | ParseException | AuthorizationException e`

https://github.com/folio-org/mod-camunda/blob/master/src/main/java/org/folio/rest/camunda/controller/advice/WorkflowControllerAdvice.java#L17

This is a reasonable error log to add instead of updating controller advice which needs more planning on exception handling.

Reply:
The exception is still being thrown.

Reply:
Okay, I will remove the exception from the error log.